ntp-proto/src/algorithm/kalman/mod.rs

```diff
@@ -1,5 +1,6 @@
 use std::{collections::HashMap, fmt::Debug, hash::Hash, time::Duration};
 
+pub use source::AveragingBuffer;
```

Comment:
I'd strongly prefer if we do not make this type public. This is basically an implementation detail.

Reply:
Fixed in 180f7e1